pkg/detectors/autoklose/autoklose.go

```diff
@@ -63,7 +66,15 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 				defer res.Body.Close()
 				if res.StatusCode >= 200 && res.StatusCode < 300 {
```

Comment:
The API returns a 200 status code for successful calls, which we can explicitly handle in this case.

Reply:
done